compiler/src/dotty/tools/dotc/core/TypeErasure.scala

```diff
@@ -892,7 +954,9 @@ class TypeErasure(sourceLanguage: SourceLanguage, semiEraseVCs: Boolean, isConst
         sigName(tp.underlying)
       case tp: WildcardType =>
         tpnme.Uninstantiated
-      case _: ErrorType | NoType =>
+      case tp: ErrorType =>
```

Comment:
Why was this case split into two? Asking just out of curiosity.

Reply:
It's an artefact of the previous encoding with null having to work around https://github.com/lampepfl/dotty/issues/18083, I can remove it.